test/e2e/tests/test_endpoint.py

```diff
@@ -224,12 +236,12 @@ def create_endpoint_test(self, sagemaker_client, xgboost_endpoint):
         assert_endpoint_status_in_sync(
             endpoint_name, reference, cfg.ENDPOINT_STATUS_CREATING
         )
-        assert k8s.wait_on_condition(reference, "ACK.ResourceSynced", "False")
+        assert k8s.wait_on_condition(reference, "ACK.ResourceSynced", "False", 3, 60)
```

Comment:
are we seeing this assert fail or just increasing timeout for everything?

this was intentional. 3 min is too long for resource to be in sync with service it should happen in 30-1min since that's the requeue time and the check above is checking the status on the server side

same is valid for all `ACK.ResourceSynced` condition waits